src/containers/genres/js/genres.js

```diff
@@ -3,6 +3,8 @@ var inputYes = document.querySelectorAll("input.yes");
 var inputNo = document.querySelectorAll("input.no");
 var label = document.querySelectorAll("label");
 var input = document.querySelectorAll("input");
+var genresChecked = 0;
+console.log(genresChecked);
```

Comment:
Is it necessary to leave all `console.log`?

Reply:
I didn't notice I left them in there. I'll delete them.